apps/nextjs-website/src/lib/webinars/webinarQuestions.ts

```diff
@@ -0,0 +1,92 @@
+import { pipe } from 'fp-ts/lib/function';
+import * as RTE from 'fp-ts/lib/ReaderTaskEither';
+import * as RA from 'fp-ts/lib/ReadonlyArray';
+import * as TE from 'fp-ts/lib/TaskEither';
+import * as R from 'fp-ts/lib/Reader';
+import * as E from 'fp-ts/lib/Either';
+import {
+  DynamoDBClient,
+  PutItemCommand,
+  QueryCommand,
+} from '@aws-sdk/client-dynamodb';
+import {
+  WebinarQuestionDynamodbCodec,
+  makeDynamodbItemFromWebinarQuestion,
+  makeWebinarQuestionFromDynamodbItem,
+  makeWebinarQuestionListQueryCondition,
+} from './dynamodb/codec';
+
+export type WebinarEnv = {
+  readonly questionLifetimeInSeconds: number;
+  readonly dynamodbClient: DynamoDBClient;
+  readonly nowDate: () => Date;
+};
+
+export type InsertWebinarQuestion = Omit<
+  WebinarQuestion,
+  'createdAt' | 'expireAt'
+>;
+
+export type WebinarQuestion = {
+  readonly webinarId: string;
+  readonly givenName: string;
+  readonly familyName: string;
+  readonly question: string;
+  readonly createdAt: Date;
+  readonly expireAt: Date;
```

Comment:
```suggestion
  readonly expiresAt: Date;
```
if you accept this suggestion, in #575 the field is already named `expiresAt`

Reply:
you can ignore this suggestion, since the field, in the end, has been named `expireAt`